base/datafmt.jl

```diff
@@ -38,16 +38,16 @@ function countlines(io::IO, eol::Char)
     nl
 end
 
-readdlm(input, T::Type; opts...) = readdlm(input, invalid_dlm, T, '\n'; opts...)
-readdlm(input, dlm::Char, T::Type; opts...) = readdlm(input, dlm, T, '\n'; opts...)
+readdlm{T}(input, ::Type{T}; opts...) = readdlm(input, invalid_dlm, T, '\n'; opts...)
```

Comment:
A 1-line function like this that just passes its arguments through is a perfect example of a case where you *don't* need to specialize on every value of `T`.